src/features/public/Donations/screens/DonationsPopup.tsx

```diff
@@ -54,7 +61,10 @@ function DonationsPopup({ onClose, project }: Props): ReactElement {
         const res = await fetch(
           `${process.env.API_ENDPOINT}/app/projects/${project.id}/paymentOptions?country=${country}`,
```

Comment:
aren't the APIs being called from central location?

Reply:
@harshvitra

Reply:
This is resolved